documentation/src/docs/getting-started.md

```diff
@@ -105,5 +107,6 @@ No more events from examples/helloStream
 ...
 ```
 
-See the [README.md](https://github.com/pravega/pravega-samples/blob/v0.3.0/pravega-client-examples/README.md) file in the pravega-client-examples for more details on running the
-    HelloWorldReader application
+See the [README.md](https://github.com/pravega/pravega-samples/blob/v0.3.2/pravega-client-examples/README.md) file in the pravega-client-examples for more details on running the
```

Comment:
You r right!.But during release (in the website) when these .md files are merged it throws 404 error in the server.....so every time its better to replace with the ongoing version.

Reply:
I think that this observation has been noted by others (see [comment in PR 2850](https://github.com/pravega/pravega/pull/2850#discussion_r211338727)). We need to find a way to use a variable in all URLs instead of a hardcoded value. Otherwise, this will be very problematic and error prone.